test/test_c10d.py

```diff
@@ -469,10 +469,9 @@ def test_common_errors(self):
             gen = c10d.rendezvous('tcp://127.0.0.1:23456?rank=0')
             next(gen)
 
-    @retry_on_address_already_in_use_error
     def test_nominal(self):
         addr = 'localhost'
-        port = common.find_free_port()
+        port = 0
```

Comment:
Thanks for adding the fix!

Curious, I saw `find_free_port` and `@retry_on_address_already_in_use_error` used in several different places. Is `test_nominal` the only victim test?